### ISSUE TITLE
UI/UX efficiency

### ISSUE THREAD

**Issue by Braintelligence** [PhieF/CarnetNextcloud#24]
When opening carnet I don't want to move my mouse to the bottom right to start a new note. I want to click on to a new sheet of paper and start typing without reloading for many seconds and then clicking around to set a title and tags and then the content.

I want to be able to operate carnet as much as possible with keyboard only.

The design should have been:
Title at top
Tags second (autocompletion)
Content last

And this should be available as instant note taking when operating the app, not only when I click something like "new note". Or at the very least put the "new note" button away from bottom right as a new card with a plus symbol in the middle or something.

**Comment by adocampo** [comment]
Well, I though I already created an issue to give some ideas of usability of the UI. I don't want to underestimate the work done, but for me, the most practical approach for a note taking application is Google Keep. I really want to stop using it, but I must admit its UX is simply awesome.

![image](https://user-images.githubusercontent.com/2266612/50728279-9902b900-1127-11e9-82fd-33471b78d5e5.png)

On the image, you can see you have all the notes, the first one is pinned, the third is shared with others. On top of everything, you have a textbox where you can enter quickly a note, which will be a popup over the main UI, so no need to go back and forth, which is extremely quick and productive.

![image](https://user-images.githubusercontent.com/2266612/50728336-52618e80-1128-11e9-947f-df504167ba39.png)

You can switch there to a list note (with the three dots menu), and other useful options (color the note, share, create a task, etc). 

For me, creating a rich text note is not useful, I would rather prefer to create quick notes and reusable lists (grocery, etc). The ability to put a link and creating a thumbnail preview (like telegram does, for example) is quite cool.

Well, I know this will probably won't be seen in Carnet as is a huge change in the UX/UI, and I'm not a developer myself so I cannot help to do it, but it would be great if developer consider some of the good job done on Google and take some ideas from them.

Anyway, I think Carnet has a lot of potential and is on the right way to succeed. ;)

**Comment by ratermir** [comment]
Carnet is very good idea, I like it and I appreciate wok already done. But the UI looks little chaotic. The "Add not" button in the right bottom corner (additionaly it is hidden by jsfx chat it it is open), another controls on the top right, even more controls on the left ...

I must agree with @adocampo - Google Keep has very good UI (Google has huge team of UX experts, so their work must be seen somewhere :) ). So I would take Keep as an inspiration. Here are some tips:
* to create / edit notes in smaller - reasonable sized - "modal-like" window (because the note is shown in this shape, it should be editted similarly)
* to collect all controls (except global) to single place - the bottom of the "modal" (edit) window is a good candidate
* I would also appreciate alternative "grid" view (similar to Colornotes android app) - but this is only "cherry on the cake" :) )
* I am not sure whether the separate title of note is necessary. First row could be a note title ... but this is not too important

I also want to stop using google (because of Google :) ), and Carnet could be a good candidate after some UI imptovements. Thank you for good work.

**Comment by locness3** [comment]
#59

**Comment by schtibal** [comment]
I think that also the "Exit" from a note is not very intuitive, nor in line with expected UI behaviour. Even though I know where it is, I still find myself looking for an 'X' in the top right of the note to save and exit.

**Comment by adocampo** [comment]
Agree, I always need to figure how to exit the note, despite I already know there is an "Exit" button on the left, where I never go in the first when looking for an exit in applications.
I would suggest let the user press "Esc" or click outside the note to exit as well which is a normal behavior nowadays
